docs/install/index.md

```diff
@@ -65,7 +65,7 @@ Indicate your preferred configuration. Then, follow the customized commands to i
   <div class="python">
     <div class="cpu">
 
-The following installation instructions have been tested on Ubuntu 14.04 and 16.04.
```

Comment:
14.04 compatibility should be kept and promised to users.

Reply:
Fixed.